contracts/penpie_adapter.sol

```diff
@@ -77,7 +77,7 @@ contract PenpieAdapter is Initializable, OwnableUpgradeable, PausableUpgradeable
     /**
      * @dev updateReward
      */
-    function updateReward() external {  _updateReward(); }
+    function updateReward() external whenNotPaused {  _updateReward(); }
```

Comment:
Please remove the 'whenNotPaused' modifier, as we now allow the normal distribution of rewards during a pause.